cv32e40x/tests/programs/corev-dv/corev_rand_instr_obi_err/test.yaml

```diff
@@ -10,4 +10,4 @@ plusargs: >
     +obi_memory_instr_random_err_enabled
     +obi_memory_instr_one_shot_err_enabled
     +gen_irq_noise
-    +buserr_sb_disabled
+    
```

Comment:
not a blocker, but you might want to change your editor configuration to not give these additions to PRs

Reply:
fixed in last commit